### PR TITLE
Ks/repl-to-vscode-changes 

### PR DESCRIPTION
**Pull request by kelsey-steven-ada**
Notes: 
* "Variables & IO" and "Conditionals" were the only two files I saw that needed repl -> VSCode changes. 
* This PR only covers "Conditionals" since Ashley rocks and is doing the repl changes in "Variables & IO" with the other updates in that file.

Changes
* On Conditionals lesson, update examples using repl formatting to use python file formatting.

### CODE REVIEW

File: 01-intro-to-python/02-conditionals.md [hunk 1]
```diff
@@ -92,19 +91,24 @@ user_input_string = input("Guess the number: ")
 
 This is where things get tricky, though!  In order to compare the user input to the random number generated, we need to convert it into a number as well.  We can use the `int` function for that, but if you pass it a badly formatted string, it will cause an error.
 
+Let's try this out, what happens when you add the following line to a python file and run it?
```
Comment (anselrognlie) [inline 01-intro-to-python/02-conditionals.md:94]: Maybe add new to reinforce this shouldn't get added to the game.py stuff the student is working on?
```suggestion
Let's try this out, what happens when you add the following line to a new python file and run it?
```

**Review by anselrognlie — APPROVED** [review]
LGTM. Just a minor suggestion at your discretion.

File: 01-intro-to-python/02-conditionals.md [hunk 2]
```diff
@@ -92,19 +91,24 @@ user_input_string = input("Guess the number: ")
...
 
 To get around this, we are going to use the string method `isnumeric`.  This will allow us to check if the string is well formatted _before_ we attempt to convert it into an integer.
 
 ```python
->>> "four".isnumeric()
-False
+print("four".isnumeric()) # Outputs `False`
 ```
 
 Note: `isnumeric` _should_ be named `is_numeric`, but Python doesn't always follow its own naming conventions everywhere.
```
Comment (anselrognlie) [inline 01-intro-to-python/02-conditionals.md:114]: Oops. Sorry I missed this. There's a typo on line 146. eachother → each other

Could you fix that while you're in the file?

**Review by anselrognlie — APPROVED** [review]
(Forgot to mention a typo)

File: 01-intro-to-python/02-conditionals.md (hunk 2 above)
Comment (kelsey-steven-ada) [reply 01-intro-to-python/02-conditionals.md:114]: Nice catch, I'll get that fixed ^_^

**Pull request merged**